2024-02-19 20:47:36,302 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 20:47:36,302 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Configure stats pid to 25888
2024-02-19 20:47:36,303 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 20:47:36,303 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 20:47:36,303 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 20:47:36,304 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 20:47:36,304 INFO    MainThread:25888 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_204736-v0n6urfs\logs\debug.log
2024-02-19 20:47:36,305 INFO    MainThread:25888 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_204736-v0n6urfs\logs\debug-internal.log
2024-02-19 20:47:36,305 INFO    MainThread:25888 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000219D2307DF0>
2024-02-19 20:47:36,305 INFO    MainThread:25888 [wandb_init.py:init():566] calling init triggers
2024-02-19 20:47:36,305 INFO    MainThread:25888 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_samples': 2, 'max_depth': 5}
2024-02-19 20:47:36,305 INFO    MainThread:25888 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 53ehprmm
2024-02-19 20:47:36,307 INFO    MainThread:25888 [wandb_run.py:_finish():1970] finishing run mc0c0z/DecisionTree/53ehprmm
2024-02-19 20:47:36,307 INFO    MainThread:25888 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-19 20:47:36,309 INFO    MainThread:25888 [wandb_run.py:_restore():2193] restore
2024-02-19 20:47:36,309 INFO    MainThread:25888 [wandb_run.py:_restore():2199] restore done
2024-02-19 20:47:40,141 INFO    MainThread:25888 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-19 20:47:40,141 INFO    MainThread:25888 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-19 20:47:40,148 INFO    MainThread:25888 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-02-19 20:47:40,249 INFO    MainThread:25888 [wandb_init.py:init():616] starting backend
2024-02-19 20:47:40,250 INFO    MainThread:25888 [wandb_init.py:init():620] setting up manager
2024-02-19 20:47:40,251 INFO    MainThread:25888 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-19 20:47:40,253 INFO    MainThread:25888 [wandb_init.py:init():628] backend started and connected
2024-02-19 20:47:40,261 INFO    MainThread:25888 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-19 20:47:40,261 INFO    MainThread:25888 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-19 20:47:40,261 INFO    MainThread:25888 [wandb_init.py:init():720] updated telemetry
2024-02-19 20:47:40,263 INFO    MainThread:25888 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-19 20:47:41,746 INFO    MainThread:25888 [wandb_run.py:_on_init():2262] communicating current version
2024-02-19 20:47:42,141 INFO    MainThread:25888 [wandb_run.py:_on_init():2271] got version response 
2024-02-19 20:47:42,141 INFO    MainThread:25888 [wandb_init.py:init():804] starting run threads in backend
2024-02-19 20:47:42,235 INFO    MainThread:25888 [wandb_run.py:_console_start():2241] atexit reg
2024-02-19 20:47:42,235 INFO    MainThread:25888 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-19 20:47:42,235 INFO    MainThread:25888 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-19 20:47:42,235 INFO    MainThread:25888 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-19 20:47:42,236 INFO    MainThread:25888 [wandb_init.py:init():847] run started, returning control to user process
2024-02-19 20:48:00,617 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 20:48:00,619 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Configure stats pid to 25888
2024-02-19 20:48:00,619 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 20:48:00,619 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 20:48:00,620 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 20:48:00,620 INFO    MainThread:25888 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 20:48:00,620 INFO    MainThread:25888 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_204800-jrctcl30\logs\debug.log
2024-02-19 20:48:00,621 INFO    MainThread:25888 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_204800-jrctcl30\logs\debug-internal.log
2024-02-19 20:48:00,621 INFO    MainThread:25888 [wandb_init.py:init():566] calling init triggers
2024-02-19 20:48:00,621 INFO    MainThread:25888 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_samples': 5, 'max_depth': 5}
2024-02-19 20:48:00,622 INFO    MainThread:25888 [wandb_init.py:init():591] re-initializing run, found existing run on stack: v0n6urfs
2024-02-19 20:48:00,624 INFO    MainThread:25888 [wandb_run.py:_finish():1970] finishing run mc0c0z/DecisionTree/v0n6urfs
2024-02-19 20:48:00,624 INFO    MainThread:25888 [jupyter.py:save_history():445] not saving jupyter history
2024-02-19 20:48:00,624 INFO    MainThread:25888 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-19 20:48:00,625 INFO    MainThread:25888 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-19 20:48:00,625 INFO    MainThread:25888 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-19 20:48:00,625 INFO    MainThread:25888 [wandb_run.py:_restore():2193] restore
2024-02-19 20:48:00,625 INFO    MainThread:25888 [wandb_run.py:_restore():2199] restore done
2024-02-19 20:48:06,194 INFO    MainThread:25888 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-19 20:48:06,196 INFO    MainThread:25888 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-19 20:48:06,203 INFO    MainThread:25888 [wandb_run.py:_footer_sync_info():3825] logging synced files
